db/views/change_in_storage_heaters_since_last_years_v01.sql

```diff
@@ -0,0 +1,35 @@
+SELECT latest_runs.id,
+       energy.*,
+       storage.temperature_adjusted_previous_year_kwh,
+       storage.temperature_adjusted_percent
+
+FROM
+  (
+    SELECT alert_generation_run_id, school_id, data.*
+    FROM alerts, alert_types, jsonb_to_record(variables) AS data(
+      previous_year_gas_kwh float,
```

Comment:
Should these not be the storage heater variables generated by this alert, rather than the gas variables? AlertEnergyAnnualVersusBenchmark generates separate sets of variables for electricity, gas and storage heaters. See `mallaig-high-school` as an example